### DISCUSSION TITLE
`pocketbase.exe update` doesn't update `CHANGELOG.md`

### DISCUSSION THREAD

**Discussion by babakfp** [pocketbase/pocketbase#3685] [Q&A] [answered]
Is it expected that `pocketbase.exe update` doesn't update `CHANGELOG.md`?

**Comment by ganigeorgiev** [comment]
The `update` command will replace only the binary.

The `CHANGELOG.md`, `LICENSE.md` or any other artifact that could be available with the zip is discarded.

**Reply by babakfp** [reply]
I think it would be nice to have them updated too. 🤷

**Reply by ganigeorgiev** [reply]
I don't think it is necessary and most of the times those files are deleted by the users anyway.

**Reply by ganigeorgiev** [reply]
Eventually we may extract the release notes and print them in the console after successful `update`, but this is definitely not a blocker or priority at the moment.

**Reply by ganigeorgiev** [reply, marked as answer]
_Since it was just a few lines of code, I've decided to add the release notes to the `update` command console output and it will be available with the next minor v0.19.3 release sometime later today._

**Reply by babakfp** [reply]
Awesome 👏
